docs/source/contributing/development.rst

```diff
@@ -66,6 +66,8 @@ For first-time contributors
      as create and enter a virtual environment. You can always re-enter
      that environment by running ``poetry shell``.
 
+   - In case you want to install extra dependencies that are defined in ```pyproject.toml``, section ``[tool.poetry.extras]`` , this can be done with e.g. ``poetry install -E jupyterlab -E gui``.
```

Comment:
```suggestion
   - In case you want to install extra dependencies that are defined in
     the ``[tool.poetry.extras]``  section of ``pyproject.toml``, this can be done by passing
     the ``-E`` flag, for example ``poetry install -E jupyterlab -E gui``.
```

Line is too long, should be broken up; plus minor improvement of wording.